src/components/SubscribeNotification/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import BellWhiteImage from 'common/icons/bellWhite.svg';
+import BellBlackImage from 'common/icons/bellBlack.svg';
+import PropTypes from 'prop-types';
+import styles from './NotificationBell.module.css';
+import cn from 'classnames';
+
+const SubscribeNotification = ({ hasSubscribed = false } = {}) => {
```

Comment:
Optional: 命名 Button 可能會比較清楚知道這是按鈕
```suggestion
const SubscribeNotificationButton = ({ hasSubscribed = false } = {}) => {
```

Reply:
已更改～順便把 `SubscribeNotificationButton` 該檔案搬到 `CompanyAndJobTitle` foler 底下